.circleci/config.yml

```diff
@@ -246,7 +246,6 @@ workflows:
                   "amplify_analytics_pinpoint_android",
                   "amplify_api_android",
                   "amplify_auth_cognito",
-                  "amplify_datastore",
```

Comment:
iOS/Android unit tests are no longer needed in this branch. Integration tests are enabled temporarily for local testing while core types are migrated.